app/routes/answer.py

```diff
@@ -0,0 +1,24 @@
+from app.server import server
+from app.controllers import answer
+from flask import render_template
+
+
+# noinspection PyUnusedLocal
+@server.route("/post/<int:post_id>/<post_title>/answer/<int:answer_id>")
+def get_answer(post_id, post_title, answer_id):
+    post_data = answer.get_answer(post_id, answer_id)
+
+    if post_data is None:
+        return render_template('notfound.html'), 404
+
+    return render_template('answer.html', post_data=post_data)
+
+
+@server.route("/post/<int:post_id>/answer/<int:answer_id>")
+def get_answer_no_post_id(post_id, answer_id):
```

Comment:
couldn't you merge this with the previous function and have two `@server.route`s?

Reply:
No, because then there would have to be an optional parameter (`post_title=None`) in the middle, and there can't be required parameters after optional parameters.